src/rdkafka.c

```diff
@@ -58,7 +58,7 @@
 #include <sys/timeb.h>
 #endif
 
-
+extern int validate_scope;
```

Comment:
This is neither thread-safe or multi-instance safe.

Reply:
Most of librdkafka APIs are not thread safe. Unless I make this into an atomic variable, this will eventually take effect when CPU cache is synced. I really don't think it's a big issue. Also this variable will most likely be set/unset once per program.